src/index.js

```diff
@@ -50,4 +52,4 @@ addEventListener('fetch', async function handleRequest(event) {
   });
   // Send the response back to the client.
   event.respondWith(response);
```

Comment:
This will need to change to `return response;` (with the following `return` removed), and same for line 29 above.

Reply:
Oh yeah! My bad haha! 😂 Fixing that now! 😄